src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">    
+    <Import Project="..\..\build\common.props" />
+    <PropertyGroup>
+        <Description>"Build-time references required to enable Razor view compilation as part of building the application."</Description>
+        <TargetFramework>netcoreapp1.0</TargetFramework>
```

Comment:
I think we would have to cross-compile the binary since we load user code in the tool. Does it work for net451 targeting projects?